include/gatekeeper_gk.h

```diff
@@ -105,6 +106,19 @@ struct gk_instance {
 	struct token_bucket_ratelimit_state back_icmp_rs;
 } __rte_cache_aligned;
 
+#define GK_MAX_BPF_FLOW_HANDLERS	(UINT8_MAX + 1)
```

Comment:
Shouldn't we use a single blank space here as we only have one ``define`` and don't need to align with others?

Reply:
Section "1.4.3. Macros" of [DPDK Coding Style](https://doc.dpdk.org/guides/contributing/coding_style.html) doesn't mention anything about it. But inspecting DPDK source, one finds many cases that use a TAB.